src/dbnode/client/options.go

```diff
@@ -426,6 +430,7 @@ func newOptions() *options {
 		hostQueueEmitsHealthStatus:              defaultHostQueueEmitsHealthStatus,
 		seriesIteratorPoolSize:                  defaultSeriesIteratorPoolSize,
 		seriesIteratorArrayPoolBuckets:          defaultSeriesIteratorArrayPoolBuckets,
+		checkedBytesPoolSize:                    defaultCheckedBytesPoolSize,
```

Comment:
I'm not sure if there are being used anywhere.

Reply:
https://github.com/m3db/m3/pull/3605/commits/02c521e4314c8a628a281e1b06d526fb6394b805